mlflow/store/tracking/abstract_store.py

```diff
@@ -694,8 +695,8 @@ def create_logged_model(
         experiment_id: str,
         name: str,
         run_id: Optional[str] = None,
-        tags: Optional[Dict[str, str]] = None,
-        params: Optional[Dict[str, str]] = None,
+        tags: Optional[ModelTag] = None,
+        params: Optional[List[ModelParam]] = None,
```

Comment:
@harupy Do we have to update FileStore or existing MlflowClient method implementations for LogModels to account for this?

Reply:
let me check

Reply:
This change comes from `FileStore`.